src/flags/var-alias/index.ts

```diff
@@ -26,6 +32,46 @@ export function getVariableAliases(
             }
             return { ...map, [codeRef]: variableName }
         },
-        variableAliasesFromConfig,
+        variableAliases,
     )
 }
+
+const addAliasesFromTypeGenerator = (
```

Comment:
it'd probably be a good idea to add some unit tests to verify this continues to work as expected

Reply:
i couldnt find any existing tests for the aliases system, do you know if there are any

Reply:
I think most of the alias tests are within the command-specific tests. In this case I could see value in a colocated unit test that validates this method specifically

Reply:
added some tests if you want to take another look